spaceship/lib/spaceship/portal/ui/select_team.rb

```diff
@@ -32,9 +32,15 @@ class UserInterface
       #   ]
 
       def select_team
-        teams = client.teams
-
-        raise "Your account is in no teams" if teams.count == 0
+        begin
+          teams = client.teams
+          raise "Your account is in no teams" if teams.count == 0
+        rescue
+          puts "No teams available on the Developer Portal"
+          puts "⚠️ you must accept an invitation to a team for it to be available"
+          puts "To learn more about teams and how to use them visit https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/ManagingYourTeam/ManagingYourTeam.html"
+          return
```

Comment:
I'm pretty fresh to Ruby, feel free to chalk it up to that. Would re-raising the exception cause the match command crash this came from again?